doc/develop/test/pytest.rst

```diff
@@ -28,7 +28,7 @@ developed as a part of Zephyr’s tree. To enable install-less operation, twiste
 they must add ``--allow-installed-plugin`` flag to twister’s call.
 
 Pytest-based test suites are discovered the same way as other twister tests, i.e., by a presence
-of testcase/sample.yaml. Inside, a keyword ``harness`` tells twister how to handle a given test.
+of test/sample.yaml. Inside, a keyword ``harness`` tells twister how to handle a given test.
```

Comment:
@nashif 

That change looks incorrect. We don't have `test.yaml` files, we have `testcase.yaml`.

I.e: https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/kernel/smp/testcase.yaml